na-toolbar.html

```diff
@@ -49,8 +49,12 @@
     </style>
     <iron-icon icon="bookmark"></iron-icon>
     <h1>Notes</h1>
-    <iron-icon icon="cloud" hidden$="[[!online]]"></iron-icon>
-    <iron-icon icon="cloud-off" hidden$="[[online]]"></iron-icon>
+    <span hidden$="[[!online]]">
+      <iron-icon icon="cloud"></iron-icon>
+    </span>
+    <span hidden$="[[online]]">
+      <iron-icon icon="cloud-off"></iron-icon>
+    </span>
```

Comment:
Ok, I can add it to iron-icon instead but this definitely qualifies as a slippery slope.